### DISCUSSION TITLE
Doc about theme sizes

### DISCUSSION THREAD

**Discussion by ygqygq2** [chakra-ui/chakra-ui#8542] [General]
### Description

https://v2.chakra-ui.com/docs/styled-system/theme#sizes

It should be like this: 

```
import { extendTheme } from '@chakra-ui/react'

const sizes = {
//  sizes: {
    ...theme.space,
    max: 'max-content',
    min: 'min-content',
    full: '100%',
    '3xs': '14rem',
    '2xs': '16rem',
    xs: '20rem',
    sm: '24rem',
    md: '28rem',
    lg: '32rem',
    xl: '36rem',
    '2xl': '42rem',
    '3xl': '48rem',
    '4xl': '56rem',
    '5xl': '64rem',
    '6xl': '72rem',
    '7xl': '80rem',
    '8xl': '90rem',
    container: {
      sm: '640px',
      md: '768px',
      lg: '1024px',
      xl: '1280px',
    },
//  },
}

const theme = extendTheme({ sizes, ...})
```

### Link to Reproduction

https://v2.chakra-ui.com/docs/styled-system/theme#sizes

### Chakra UI Version

v2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by cidkumagai** [comment]
Hi @ygqygq2 Can I work on this issue?

**Comment by veloware** [comment]
seems pointless trying to have docs in sync with actual code, either the docs need to either - 

- inject the code file itself
- be a brief _example_ , not a perfect replica, of the actual code

**Comment by segunadebayo** [comment]
That page is v2. v2 docs and the theme are frozen.

In v3, sizes live under `theme.tokens.sizes`. https://chakra-ui.com/docs/theming/customize-theme
